client/my-sites/plan-features/index.jsx

```diff
@@ -6,6 +6,7 @@ import { connect } from 'react-redux';
 import classNames from 'classnames';
 import noop from 'lodash/noop';
 import page from 'page';
+import lodashMap from 'lodash/map';
```

Comment:
I see your concern, but it's pretty clear from the import and we have other usages in the code.